CONTRIBUTING.md

```diff
@@ -79,10 +76,9 @@ Header includes must be [levelized](./Builds/levelization).
 
 ## Pull requests
 
-Pull requests must target the `develop` branch.[^1]
+In general, pull requests use `develop` as the base branch.
 
-[^1]: There are exceptions to this policy for hotfixes, but no one consulting
-  this document will be in that situation.
+(Hotfixes are an exception.)
 
 Changes to pull requests must be added as new commits.
 You may **never force push a branch in a pull request** (e.g. after a rebase).
```

Comment:
But is it possible to not-force-push a branch? In cases where we need to clean up the commit history, I feel force push is a last resort.

Reply:
ok, these are suggestions. I read the below paragraph 👍